src/scenario/scenario_events_parameter_data.c

```diff
@@ -9,6 +9,13 @@
 #include "scenario/custom_messages.h"
 #include "scenario/scenario.h"
 
+#ifdef _MSC_VER
+// Of course MSVC is the only compiler that doesn't have POSIX strcasecmp...
+#include <mbstring.h>
+#else
+#include <strings.h>
+#endif
+
```

Comment:
Please don't do this. Don't use compiler-specific `defines` outside of files inside the `platform` folder.

Besides, you won't really need this as you'll see below.

Reply:
Removed.